log mismatch with success message on generate.

I pressed the generate button from the formgrader. 
At the top of the window, it says "success". 

![image](https://user-images.githubusercontent.com/393899/28242335-635c436e-696e-11e7-9956-f3f2e1365690.png)

But, if you look at the log output, it clearly bombed. 
full log output if you click on "Details" below.

<details>

```

[WARNING] Config option `template_file` not recognized by `NotebookExporter`.
[WARNING] Config option `template_path` not recognized by `NotebookExporter`.
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.aux -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.aux
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.fdb_latexmk -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.fdb_latexmk
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.fls -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.fls
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.log -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.log
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.out -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.out
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.pdf -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.pdf
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.tex -> /Users/khuff/repos/pvt-npre412/release/./hw05/hw05.tex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/0AB0737392C42FEDA13B9F3F0F9B629C6315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/0AB0737392C42FEDA13B9F3F0F9B629C6315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/262BDCC3EE2CF7A56E9AFE7570B3E8FE6315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/262BDCC3EE2CF7A56E9AFE7570B3E8FE6315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/551F6190F7185C1E097E13B972DAFFC76315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/551F6190F7185C1E097E13B972DAFFC76315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/6D9D384696374F234E2B04BB95908B606315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/6D9D384696374F234E2B04BB95908B606315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/774774A8D6DF307FE9A8BE1E79AD33C16315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/774774A8D6DF307FE9A8BE1E79AD33C16315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/994C9D163D75AA88222829793A3C56A16315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/994C9D163D75AA88222829793A3C56A16315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/9D626CA2C24F94164A510DA4C56C93D86315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/9D626CA2C24F94164A510DA4C56C93D86315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/AF68410752A0AE38B91D4938C4A890286315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/AF68410752A0AE38B91D4938C4A890286315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/D56B0D8658FF0AE47FBC36D2F7CDB1076315FB99A715581E434B7CBB305AA31C.pygtex -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/D56B0D8658FF0AE47FBC36D2F7CDB1076315FB99A715581E434B7CBB305AA31C.pygtex
[INFO] Copying /Users/khuff/repos/pvt-npre412/source/./hw05/_minted-hw05/default.pygstyle -> /Users/khuff/repos/pvt-npre412/release/./hw05/_minted-hw05/default.pygstyle
[INFO] Updating/creating assignment 'hw05': {}
[INFO] Converting notebook /Users/khuff/repos/pvt-npre412/source/./hw05/hw05.ipynb
[ERROR] There was an error processing assignment: /Users/khuff/repos/pvt-npre412/source/./hw05
[ERROR] Traceback (most recent call last):
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbgrader/converters/base.py", line 288, in convert_notebooks
        self.convert_single_notebook(notebook_filename)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbgrader/converters/base.py", line 244, in convert_single_notebook
        output, resources = self.exporter.from_filename(notebook_filename, resources=resources)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/exporters/exporter.py", line 172, in from_filename
        return self.from_file(f, resources=resources, **kw)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/exporters/exporter.py", line 190, in from_file
        return self.from_notebook_node(nbformat.read(file_stream, as_version=4), resources=resources, **kw)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/exporters/notebook.py", line 31, in from_notebook_node
        nb_copy, resources = super(NotebookExporter, self).from_notebook_node(nb, resources, **kw)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/exporters/exporter.py", line 132, in from_notebook_node
        nb_copy, resources = self._preprocess(nb_copy, resources)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/exporters/exporter.py", line 309, in _preprocess
        nbc, resc = preprocessor(nbc, resc)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/preprocessors/base.py", line 47, in __call__
        return self.preprocess(nb,resources)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbgrader/preprocessors/clearsolutions.py", line 130, in preprocess
        nb, resources = super(ClearSolutions, self).preprocess(nb, resources)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbconvert/preprocessors/base.py", line 70, in preprocess
        nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
      File "/Users/khuff/anaconda/lib/python3.5/site-packages/nbgrader/preprocessors/clearsolutions.py", line 149, in preprocess_cell
        "Solution region detected in a non-solution cell; please make sure "
    RuntimeError: Solution region detected in a non-solution cell; please make sure all solution regions are within solution cells.
    
[WARNING] Removing failed assignment: /Users/khuff/repos/pvt-npre412/release/hw05
[ERROR] There was an error processing assignment 'hw05' for student '.'

```

</details>

--- a/nbgrader/converters/base.py
+++ b/nbgrader/converters/base.py
@@ -336,5 +336,10 @@ class BaseConverter(LoggingConfigurable):
                 msg = (
                     "Please see the error log ({}) for details on the specific "
                     "errors on the above failures.".format(self.logfile))
-                self.log.error(msg)
-                raise NbGraderException(msg)
+            else:
+                msg = (
+                    "Please see the the above traceback for details on the specific "
+                    "errors on the above failures.")
+
+            self.log.error(msg)
+            raise NbGraderException(msg)

